Uplift/Views/GymDetail/Hours/GymDetailHoursCell.swift

```diff
@@ -75,18 +76,41 @@ class GymDetailHoursCell: UICollectionViewCell {
         hoursTableView.register(GymHoursHeaderView.self, forHeaderFooterViewReuseIdentifier: GymHoursHeaderView.identifier)
         hoursTableView.delegate = self
         hoursTableView.dataSource = self
+        hoursTableView.translatesAutoresizingMaskIntoConstraints = false
         contentView.addSubview(hoursTableView)
 
+        // TODO: - Fix layout to match designs
         dividerView.backgroundColor = .gray01
+//        dividerView.backgroundColor = .clear
+        dividerView.translatesAutoresizingMaskIntoConstraints = false
         contentView.addSubview(dividerView)
     }
 
     private func getTableViewHeight() -> CGFloat {
+        if hours.isEmpty() {
+            return 0.0
+        }
+
+        var height = LayoutConstants.hoursTableViewBottomRowHeight
+
+        if isDisclosed {
+         height += CGFloat(hours.getNumHoursLines() - 1) * LayoutConstants.hoursTableViewRowHeight
+        }
+
+        return height
+    }
+
+    public static func getHeight(isDisclosed: Bool, numLines: Int) -> CGFloat {
+        var height = LayoutConstants.hoursTableViewBottomRowHeight
         if isDisclosed {
-            return CGFloat(hours.getNumHoursLines() - 1) * LayoutConstants.hoursTableViewRowHeight + LayoutConstants.hoursTableViewBottomRowHeight
-        } else {
-            return LayoutConstants.hoursTableViewBottomRowHeight
+            height += CGFloat(numLines - 1) * LayoutConstants.hoursTableViewRowHeight
         }
+        height += GymDetailConstraints.verticalPadding
```

Comment:
Is the reason this is written in multiple lines so that it's easier to read?

Reply:
yeah its easier to read and it lines up with the vertical lines of setting constraints.